src/handlers/processors.ts

```diff
@@ -6,6 +6,7 @@ import { nullHandler } from "./shared";
 import { handleComment, issueClosedCallback } from "./comment";
 import { checkPullRequests } from "./assign/auto";
 import { createDevPoolPR } from "./pull-request";
+import { incentiveContribution } from "./payout";
```

Comment:
We should rename this function to `incentivizeComments` to be more precise with its purpose?